satpy/hrit_msg_tutorial.ipynb

```diff
@@ -32,19 +32,21 @@
     "from satpy.resample import get_area_def\n",
```

Comment:
<p>Would you mind changing this to say:</p><p> </p><pre class="ql-syntax" spellcheck="false">You may see warnings due to deprecated pyproj features and when numpy/dask process NaNs values. NaNs are used to mark invalid pixels and space pixels (like you'll see in a full disk image).
</pre><p> </p><p>This way if we fix the warnings this doesn't sound as certain. You could also put a backtick (`) on either side of the warnings line so it becomes literal text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pytroll/pytroll-examples/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>